workflow/schemas/config.schema.yaml

```diff
@@ -21,6 +21,16 @@ properties:
       Path to resources.yaml file
     format: uri-reference
 
+  pacbio_alignment:
```

Comment:
Is this and `ont_alignment` related to the artifact filter?

Reply:
Yes and no. With the artifact filter i needed to update the cnv_sv module, and in the new version there are some rules for long-read added and by that a new format possible format for the units file. The `pacbio_alignment` and `ont_alignment` are used to separate a [short-read from a long-read units.tsv](https://github.com/hydra-genetics/cnv_sv/blob/d2a38e361550a335600ef41a50759f54b96bdb50/workflow/rules/common.smk#L36). 
Jonas and Padraic are working on a solution where Hydra can figure out what type of sequencing are used by itself. But for now its the `_alignment` variables that determines which columns should be used, so I set those to False by default for Poppy since it is designed for short-read.